frontend/src/components/common/header/header.tsx

```diff
@@ -97,74 +68,66 @@ const Header: React.FC = () => {
-                        <Link
-                          to={AppRoute.ROOT}
-                          className={styles.link}
-                          onClick={handleUserExit}
-                        >
-                          Exit
-                        </Link>
-                      </li>
-                    </ul>
-                  </div>
-                </Popuper>
+                  renderContent={(close): JSX.Element => (
+                    <div
+                      className={styles.dropDown}
+                    >
+                      <ul className={styles.dropDownList}>
+                        <li className={styles.dropDownListItem}>
+                          <Link
+                            to={`${AppRoute.USERS}/${user?.id}`}
+                            className={styles.link}
+                            onClick={close}
+                          >
+                            My Profile
+                          </Link>
+                        </li>
+                        <li className={styles.dropDownListItem}>
+                          <Link
+                            to={AppRoute.PODCASTS_EDIT}
+                            className={styles.link}
+                            onClick={close}
+                          >
+                            + Create Podcast
+                          </Link>
+                        </li>
+                        <li className={styles.dropDownListItem}>
+                          <Link
+                            to={AppRoute.ROOT}
+                            className={styles.link}
+                            onClick={handleUserExit(close)}
+                          >
+                            Exit
+                          </Link>
+                        </li>
+                      </ul>
+                    </div>)}
+                />
               </div>
             </div>
             <Popuper
               trigger={
                 <div
-                  className={getAllowedClasses(
-                    styles.burgerMenu,
-                    isBurgerVisible && styles.active,
-                  )}
-                  onClick={handleBurgerToggle}
-                  ref={burgerRef}
+                  className={styles.burgerMenu}
                 >
                   <span></span>
                   <span></span>
                   <span></span>
                 </div>
               }
-            >
-              <ul
+              renderContent={(close): JSX.Element => (<ul
                 className={getAllowedClasses(
                   styles.navigation,
                   styles.burgerNav,
-                  isBurgerVisible && styles.active,
                 )}
               >
                 <li className={styles.navigationItem}>
-                  <Link to={AppRoute.ROOT} className={styles.link}>
+                  <Link to={AppRoute.ROOT} className={styles.link} onClick={close}>
```

Comment:
```suggestion
                  <Link to={AppRoute.ROOT} className={styles.link}>
```
we don't want to add it for each item